src/BaseUser.php

```diff
@@ -12,6 +13,12 @@ class BaseUser
     private ?string $displayName;
     private ?string $mail;
     private ?string $onPremisesSamAccountName;
+    private ?string $surName;
```

Comment:
I think its one word:

```suggestion
    private ?string $surname;
```